data/json/items/armor.json

```diff
@@ -836,6 +836,7 @@
     "weight": 60,
     "volume": 0,
     "price": 5000,
+    "price_postapoc": 700,
```

Comment:
I'm a little late with this, but this addition was unnecessary as you shouldn't find this in a normal game, ever.